lib/ProductOpener/Products.pm

```diff
@@ -2954,6 +2955,32 @@ or "slack_CHANNEL_NAME" (B<warning> currently channel name is ignored, we post t
 
 =cut
 
+sub removeEmailValues ($product_ref) {
+
+	# Iterate over the product fields
+	foreach my $field (keys %{$product_ref->{product}}) {
```

Comment:
There's no "product" field in the product object, the fields are at the root of the hash.

```suggestion
	foreach my $field (keys %$product_ref) {
```